vaadinfrontend/frontend/styles/views/project/project-view.css

```diff
@@ -1,16 +1,16 @@
-/* Subdivides the project-view-page into 4 columns and rows with equal spacing on which the components can be allocated*/
```

Comment:
Thanks to the board solution this class can be removed. Make sure to also remove all the imports of this class (There should be 1 on each page)